editor/src/clj/editor/outline_view.clj

```diff
@@ -134,6 +134,7 @@
                 :child-error? (boolean (some :child-error? data))
                 :child-overridden? (boolean (some :child-overridden? data))
                 :hideable? (contains? outline-name-paths (rest node-outline-key-path))
+                :hidden-parent? (scene-visibility/hidden-outline-key-path? hidden-node-outline-key-paths node-outline-key-path)
```

Comment:
Do we have a preference on boolean naming? I am leaning towards using a question mark on predicate methods only, but I don't mind either way. Removing `?` also feels inadequate in some cases, so having a general rule will make the decision easier.

Reply:
Yes, these days we tend to reserve the `?` suffix for predicates, and often prefixing booleans with words like `is-`, `has-`, `should-`, and `will-` when appropriate.

I don't really mind us following the convention of the surrounding code like this, though.